tests/benchmark/results.md

```diff
@@ -61,13 +61,15 @@ For Machine types: n2-standard-4
 
 #### Results post optimization to load files from GCS to Bigquery using BigqueryHook
 
-|Dataset                                    |Size | Duration(seconds)  |
-|-------------------------------------------|-----|--------------------|
-|covid_overview/covid_overview_10kb.csv     |10 KB| 13.57           |
-|tate_britain/artist_data_100kb.csv         |100KB| 16.10          |
-|imdb/title_ratings_10mb.csv                |10MB | 19.40         |
-|stackoverflow/stackoverflow_posts_1g.ndjson|1GB  | 30.26          |
-|trimmed/pypi/*                             |5GB  | 59.90          |
+| Dataset                                                                  | Size | Duration(seconds) |
+|--------------------------------------------------------------------------|------|--------|
+| covid_overview/covid_overview_10kb.csv                                   | 10 KB | 13.57  |
+| tate_britain/artist_data_100kb.csv                                       | 100KB | 16.10  |
+| imdb/title_ratings_10mb.csv                                              | 10MB | 19.40  |
+| stackoverflow/stackoverflow_posts_1g.ndjson                              | 1GB  | 30.26  |
+| trimmed/pypi/*                                                           | 5GB  | 59.90  |
+| gs://astro-sdk/benchmark/trimmed/stackoverflow/10gb/ - 10 Files 1gb each | 10GB |1.94min|
```

Comment:
is this a new dataset? May be we should add it to the list in Dataset.md

Reply:
Why is this a new dataset..?
Should we also run the benchmark for the other databases to use it, so we can have a fair comparison?

Reply:
@tatiana 
1. invalid rows with null values 
2. There was a type mismatch as well